conversation_reconstruction_dataflow/conversation_reconstruction/README.md

```diff
@@ -0,0 +1,17 @@
+# Conversation Constructor for Wikipedia Talk Pages
+
+This is the codebase for the dataflow pipeline working on constructing conversations from Wikipedia Talk Pages.
+
+Run it with 'python dataflow_main.py --setup_file ./setup.py', you can select a week in 2001 to 2017 to process, using parameters 'week' and 'year'. Revisions from the same page must be processed sequentially in temporal order.
```

Comment:
Cool. Give an actual command line example that works :) 

```
python dataflow_main.py \
  --setup_file ./setup.py \
  --week ???
```

That way things like the format of the week value are clearer. :)